src/Bridge/Symfony/Bundle/Resources/config/doctrine_orm.php

```diff
@@ -0,0 +1,132 @@
+
+        ->set('api_platform.doctrine.orm.item_data_provider')
+            ->abstract()
+            ->args([ref('doctrine'), ref('api_platform.metadata.property.name_collection_factory'), ref('api_platform.metadata.property.metadata_factory'), tagged_iterator('api_platform.doctrine.orm.query_extension.item')])
+
+        ->set('api_platform.doctrine.orm.subresource_data_provider')
+            ->abstract()
+            ->args([ref('doctrine'), ref('api_platform.metadata.property.name_collection_factory'), ref('api_platform.metadata.property.metadata_factory'), tagged_iterator('api_platform.doctrine.orm.query_extension.collection'), tagged_iterator('api_platform.doctrine.orm.query_extension.item')])
+
+        ->set('api_platform.doctrine.orm.default.collection_data_provider', CollectionDataProvider::class)
+            ->parent('api_platform.doctrine.orm.collection_data_provider')
+            ->tag('api_platform.collection_data_provider')
+
+        ->set('api_platform.doctrine.orm.default.item_data_provider', ItemDataProvider::class)
+            ->parent('api_platform.doctrine.orm.item_data_provider')
+            ->tag('api_platform.item_data_provider')
+
+        ->set('api_platform.doctrine.orm.default.subresource_data_provider', SubresourceDataProvider::class)
+            ->parent('api_platform.doctrine.orm.subresource_data_provider')
+            ->tag('api_platform.subresource_data_provider')
+
+        ->set('api_platform.doctrine.orm.search_filter', SearchFilter::class)
+            ->abstract()
+            ->args([ref('doctrine'), null, ref('api_platform.iri_converter'), ref('api_platform.property_accessor'), ref('logger')->ignoreOnInvalid(), '$identifiersExtractor' => ref('api_platform.identifiers_extractor.cached')->ignoreOnInvalid(), '$nameConverter' => ref('api_platform.name_converter')->ignoreOnInvalid()])
+        ->alias(SearchFilter::class, 'api_platform.doctrine.orm.search_filter')
+
+        ->set('api_platform.doctrine.orm.order_filter', OrderFilter::class)
+            ->abstract()
+            ->args([ref('doctrine'), null, '%api_platform.collection.order_parameter_name%', ref('logger')->ignoreOnInvalid(), '$nameConverter' => ref('api_platform.name_converter')->ignoreOnInvalid()])
+        ->alias(OrderFilter::class, 'api_platform.doctrine.orm.order_filter')
+
+        ->set('api_platform.doctrine.orm.range_filter', RangeFilter::class)
+            ->abstract()
+            ->args([ref('doctrine'), null, ref('logger')->ignoreOnInvalid(), '$nameConverter' => ref('api_platform.name_converter')->ignoreOnInvalid()])
+        ->alias(RangeFilter::class, 'api_platform.doctrine.orm.range_filter')
+
+        ->set('api_platform.doctrine.orm.date_filter', DateFilter::class)
+            ->abstract()
+            ->args([ref('doctrine'), null, ref('logger')->ignoreOnInvalid(), '$nameConverter' => ref('api_platform.name_converter')->ignoreOnInvalid()])
```

Comment:
The `ref()` function has been renamed to `service()` in 5.1. We need to declare a `service()` proxy function that delegates to the right function internally depending on existence (and remove that proxy when support for symfony/dependency-injection 4.4 gets removed in the future).

Reply:
Done, thx for you help